src/main/java/com/ays/AysApplicationEventListener.java

```diff
@@ -0,0 +1,24 @@
+package com.ays;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * A listener that logs a message when the AYS application is ready to start.
+ */
+@Slf4j
+@Configuration
+class AysApplicationEventListener implements ApplicationListener<ApplicationReadyEvent> {
```

Comment:
may i ask why you implemented this just for the sake of logging that the application is up and running?

Reply:
I have written this class to add a log indicating that the application has been started in addition to the existing logs.